docker/entry-point-poetry-jupyter.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+# stty erase ^h
+poetry run jupyter notebook --ip=0.0.0.0 &
```

Comment:
Shouldn't this be 8.8.8.8?

Reply:
No, 0.0.0.0 means listen on all addresses

Reply:
Got it!